src/main/java/nl/tudelft/cse1110/andy/result/ResultBuilder.java

```diff
@@ -253,6 +253,14 @@ public Result build() {
             GradeWeight weights = GradeWeight.fromConfig(ctx.getRunConfiguration().weights());
 
             int finalGrade = calculateFinalGrade(grades, weights);
+
+            if (ctx.getExternalProcess() != null) {
+                String errMsg = ctx.getExternalProcess().getErr();
+                if (errMsg != null) {
```

Comment:
This can be interpreted in 2 ways - the process exited with an exit code different than 0, or printed errors to stderr. Do we care about messages in stderr if the process exits with exit code 0 regardless?

Reply:
I made it so that error messages are ignored if they don't cause the process to crash. Some programs print warnings to stderr and we might want to ignore those.